app/test/task/cloudcompute/googlecloudcompute_test.dart

```diff
@@ -26,8 +26,14 @@ void main() {
       // Create CloudCompute instance
       final gce = createGoogleCloudCompute(
         project: envConfig.googleCloudProject!,
+        // NOTE: It's important to have a Cloud NAT for the default network.
+        //       This is probably only reasonable to do in a test project.
```

Comment:
Not sure: why only in the test project?

Reply:
We also have it in production project... but we can't test in that project :see_no_evil: